app/mixins/event-wizard.js

```diff
@@ -98,7 +98,7 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
 
       // model.tax is set in basic-detail step to workaround issue #4385
       let tax = this.model.tax || data.tax;
-      if (tax) {
+      if (tax && tax.name) {
```

Comment:
@pc-beast: previously it was added because, the tax were created at the time of event create (with empty names). so later if you go to create tax info, and it doesn't found the name, the modal will pop up as it will know that tax need to be created (edited in a more technical sense). But in tax refactor I have changed that flow because it was causing bugs. But kept this part, because it might cause issue for few edge cases that are off the eyes now. FYI @mariobehling